frontend/src/app/survey/create/create-survey-dialog/create-survey-dialog.component.html

```diff
@@ -2,15 +2,20 @@
     <div class="title">
         <mat-form-field class="form-field">
             <mat-label>{{ "survey.title" | translate }}</mat-label>
-            <input matInput class="input" [(ngModel)]="title" name="title">
+            <input matInput  [(ngModel)]="title" name="title">
         </mat-form-field>
         <mat-form-field class="form-field">
             <mat-label>{{ "survey.description" | translate }}</mat-label>
-            <textarea matInput class="input" [(ngModel)]="description" name="description"></textarea>
+            <textarea matInput [(ngModel)]="description" name="description"></textarea>
         </mat-form-field>
     </div>
 </div>
-<app-response-overview></app-response-overview>
+<div class="responses">
+    <app-response class="response" [mode]="'create'" (timeslotCreated)="createTimeSlot($event)"></app-response>
```

Comment:
Maybe we should discuss this again. Having the create card at the end would be problematic because the user wouldn't see this when there a lot timeslots. A solution might be to add the new created cards upfront and not on the end

Reply:
The cards should probably be ordered by the start date in the end. So I think we can keep it like this.